mantis-control-plane/mantis-control-plane-dynamodb/src/main/resources/dynamodb.properties

```diff
@@ -0,0 +1,8 @@
+# mantis.ext.dynamodb.store.table=mantis-dynamodb-kv
```

Comment:
This is an example file, but since the config framework has defaults I left them commented out.